src/quantity-selector/index.html

```diff
@@ -0,0 +1 @@
+<input type="string" onchange="{ quantityHandler }" value="{ props.value || $quantitySelector.value }" min="1">
```

Comment:
Can just access `state.value` rather than `$quantitySelector.value` if you choose to get rid of the alias